<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>serUpdateClass</class>
 <widget class="QMainWindow" name="serUpdateClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>632</width>
    <height>261</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>serUpdate</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>串口列表</string>
    </property>
   </widget>
   <widget class="QComboBox" name="serList">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="serConnect">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>30</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>连接串口</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openFile">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>30</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>打开文件</string>
    </property>
   </widget>
   <widget class="QLabel" name="bootloadVer">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>271</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>bootload版本号</string>
    </property>
   </widget>
   <widget class="QLabel" name="deviceID">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>281</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>芯片出场ID</string>
    </property>
   </widget>
   <widget class="QLabel" name="downloadStatus">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>271</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>未开始</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="downloadProgress">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>80</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="upLoad">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>150</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>开始下载</string>
    </property>
   </widget>
   <widget class="QLabel" name="fileRoute">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>581</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>文件未载入</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="serUpdate.qrc"/>
 </resources>
 <connections/>
</ui>
